tools/build/homebrew/lanraragi

```diff
@@ -20,6 +20,9 @@ fi
 if [[ ${LRR_DATA_DIRECTORY} == "" ]]; then
```

Comment:
We did merge this already, so it's weird to see it pop up again -- A rebase of the branch would be most welcome.

Reply:
I merged it again